### PR TITLE
Added the policy es_model helps to unique policy data

### PR DESCRIPTION
**Pull request by athiruma**
## Type of change
Note: Fill **x** in []
- [ ] bug
- [ ] enhancement
- [ ] documentation
- [x] dependencies

## Description
Creating PolicyEsModal helps to create the unique data to upload to the elastic_search

### CODE REVIEW

File: cloud_governance/common/elasticsearch/modals/policy_es_data.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    resource_id: str
+    user: str
+    dry_run: str
+    name: str
+    region_name: str
+    public_cloud: str
+    action_days: str
+    skip_policy: str
+    cost: float = ''
+    savings: float = ''
```
Comment (ebattat) [inline cloud_governance/common/elasticsearch/modals/policy_es_data.py:20]: yearly_saving

**Comment by ebattat** [comment]
@athiruma there is conflict, pls fix it

File: cloud_governance/common/elasticsearch/modals/policy_es_data.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    SnapshotDate: str = datetime.now(timezone.utc).date().__str__()
+
+    # Specific Policy Attributes
+    volume_size: int = ''
+    instance_type: str = ''
+    launch_time: str = ''
+    running_days: int = ''
+    create_date: str = ''
+
+    def __post_init__(self):
```
Comment (ebattat) [inline cloud_governance/common/elasticsearch/modals/policy_es_data.py:37]: What the purpose of post init, why not to use init ?

File: cloud_governance/common/utils/utils.py [hunk 3]
```diff
@@ -130,3 +131,17 @@ def get_start_and_end_datetime(days: int) -> [datetime, datetime]:
         end_date = datetime.utcnow()
         start_date = end_date - timedelta(days=days)
         return start_date, end_date
+
+    @staticmethod
+    def convert_to_title_case(snake_case: str):
+        """
+        This method convert lower case to title case
```
Comment (ebattat) [inline cloud_governance/common/utils/utils.py:138]: converts

File: cloud_governance/common/elasticsearch/modals/policy_es_data.py [hunk 4]
```diff
@@ -0,0 +1,52 @@
+from datetime import datetime, timezone
+from dataclasses import dataclass, asdict
+
+from cloud_governance.common.utils.utils import Utils
+
+
+@dataclass
+class PolicyEsMetaData(dict):
+
```
Comment (ebattat) [inline cloud_governance/common/elasticsearch/modals/policy_es_data.py:9]: can u add the class purpose ?

File: cloud_governance/common/elasticsearch/modals/policy_es_data.py [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+    # Specific Policy Attributes
+    volume_size: int = ''
+    instance_type: str = ''
+    launch_time: str = ''
+    running_days: int = ''
+    create_date: str = ''
+
+    def __post_init__(self):
+        """
+        This method initialize the IndexId
```
Comment (ebattat) [inline cloud_governance/common/elasticsearch/modals/policy_es_data.py:39]: initializes

File: cloud_governance/common/utils/utils.py [hunk 6]
```diff
@@ -130,3 +131,17 @@ def get_start_and_end_datetime(days: int) -> [datetime, datetime]:
         end_date = datetime.utcnow()
         start_date = end_date - timedelta(days=days)
         return start_date, end_date
+
+    @staticmethod
+    def convert_to_title_case(snake_case: str):
```
Comment (ebattat) [inline cloud_governance/common/utils/utils.py:136]: if this method is only for testing purpose, I think it should be under test directory

**Comment by athiruma** [comment]
I have resolved the comments.

File: cloud_governance/common/utils/utils.py (hunk 6 above)
Comment (ebattat) [reply cloud_governance/common/utils/utils.py:136]: @athiruma, WDYT ?

**Review by ebattat — APPROVED** [review]
/LGTM

**Pull request merged**